fiaas_deploy_daemon/crd/crd_resources_syncer_apiextensionsv1.py

```diff
@@ -40,8 +43,17 @@ def _create_or_update(kind, plural, short_names, group, schema_properties):
         name = "%s.%s" % (plural, group)
         metadata = ObjectMeta(name=name)
         names = CustomResourceDefinitionNames(kind=kind, plural=plural, shortNames=short_names)
-        schema = CustomResourceValidation(openAPIV3Schema=JSONSchemaProps(type="object", properties=schema_properties))
-        version_v1 = CustomResourceDefinitionVersion(name="v1", served=True, storage=True, schema=schema)
+        if kind == "Application":
+            open_apiv3_schema = JSONSchemaPropsStatusEnabled(type="object", properties=schema_properties)
+            schema = CustomResourceValidation(openAPIV3Schema=open_apiv3_schema)
+            custom_resource_subresources = CustomResourceSubresources(status=CustomResourceSubresourceStatusEnabled())
+            version_v1 = CustomResourceDefinitionVersion(name="v1", served=True, storage=True, schema=schema,
+                                                         subresources=custom_resource_subresources)
+        else:
+            open_apiv3_schema = JSONSchemaProps(type="object", properties=schema_properties)
+            schema = CustomResourceValidation(openAPIV3Schema=open_apiv3_schema)
+            version_v1 = CustomResourceDefinitionVersion(name="v1", served=True, storage=True, schema=schema)
+
```

Comment:
It's not that easy because the JSONSchemaProps changed also 😅. As you can see the `Application` is using the `JSONSchemaPropsStatusEnabled` while the ApplicationStatus is using the `JSONSchemaProps` 😔

Reply:
Another option will be to send the JsonSchema object also 😄